src/datepicker/datepicker.js

```diff
@@ -183,24 +183,26 @@ export default class Datepicker extends React.Component<
               }
               {...popoverProps}
             >
-              <InputComponent
-                aria-disabled={this.props.disabled}
-                aria-label={
-                  this.props['aria-label'] || locale.datepicker.ariaLabel
-                }
-                error={this.props.error}
-                aria-labelledby={this.props['aria-labelledby']}
-                aria-describedby={this.props['aria-describedby']}
-                aria-required={this.props.required || null}
-                disabled={this.props.disabled}
-                value={this.formatDisplayValue(this.props.value)}
-                onFocus={this.open}
-                onBlur={this.handleInputBlur}
-                onKeyDown={this.handleKeyDown}
-                placeholder={this.props.placeholder}
-                required={this.props.required}
-                {...inputProps}
-              />
+              <span>
```

Comment:
What's the span wrapper is for?

Reply:
The `Input` component does not implement ref forwarding